{{ cookiecutter.package_name }}/.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,104 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: Python Tests
+
+on: [push, pull_request]
+
+jobs:
+
+  # Set up matrix to run tox tests across lists of os, python version, and tox environment
+  matrix_tests:
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"  # github actions doesn't yet support this natively so add here
```

Comment:
This incantation doesn't work anymore. I tried and failed in astropy/astropy#11044

Reply:
oh bother. looks like there's a new option on the marketplace that might do the trick: https://github.com/marketplace/actions/ci-skip-action. it's 3rd party, though. the way forward for astropy as a whole for now may be to fork that and modify as needed for our purposes. for now i'll remove these from the template until we have a workable solution.